contracts/mapping-token/contracts/darwinia/DarwiniaMappingTokenFactory.sol

```diff
@@ -85,12 +94,14 @@ contract DarwiniaMappingTokenFactory is Initializable, Ownable {
         tokenToInfo[token] = TokenInfo(eventReceiver, tokenType, backing, source);
 
         (bool encodeSuccess, bytes memory encoded) = DISPATCH_ENCODER.call(
-            abi.encodePacked(eventReceiver, bytes4(keccak256("registered(bytes4,uint32,string,string,uint8,address,address)")),
+            abi.encodePacked(eventReceiver, bytes4(keccak256("token_register_response()")),
```

Comment:
Add comments, who it the eventReceiver?

For encoding messages and confirm to provide message delivery proof.